lib/katello.rb

```diff
@@ -12,6 +12,7 @@
 require "rest_client"
 require "anemone"
 require "securerandom"
+require "strong_parameters"
```

Comment:
seems it it related to [this issue](http://projects.theforeman.org/issues/12047), though i agree with you. in general we shouldn't need to manually require `strong_parameters`.